hspy_bundle/NSIS_installer_script.nsi

```diff
@@ -307,7 +307,7 @@ SectionIn RO
 	File /r "${WINPYTHON_PATH}\*"
 	${If} $InstMode = 2
 	; Create right-click context menu entries for Hyperspy Here
-		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" install_hyperspy_here & exit"'
+		Exec 'cmd.exe /C ""${APP_INSTDIR}\WinPython Command Prompt.exe" jupyter-context_menu-add & exit"'

```

Comment:
Just as a nitpick, I always translate underscores as spaces, so I read this as "jupyter-context menu-add".